modules/python/python-grammar.ym

```diff
@@ -135,13 +135,7 @@ python_dd_custom_options
         ;
 
 python_dd_custom_option
-        : string string_or_number
-        {
-          python_dd_set_option(last_driver, $1, $2);
-          free($1);
-          free($2);
-        }
-	| string LL_ARROW string_or_number
+        : string optional_arrow string_or_number
```

Comment:
This also removes the actual setting of options, I guess it is readded by a subsequent commit. Would be nicer with not removing that.

Reply:
Actually it does not, just one of the two setter rule become unused. However, this was solved differently with the `PythonOptions` class, so it is kinda "fixed".